contents/blog/posthog-vs-mixpanel.mdx

```diff
@@ -17,361 +17,218 @@ import { ComparisonTable } from 'components/ComparisonTable'
+PostHog autocaptures data, so you don't need to waste energy instrumenting basic events. You can choose between light and dark mode themes. Features like our custom SQL query builder, keyboard shortcuts, and command palette, cater to power users. You can setup PostHog and start paying us without ever speaking to a sales rep, though our [customer success team](/contact-sales) is always available to answer your questions.
 
-### 3. We're an open book
+### 3. We're open source and transparent
 
-PostHog is [built on transparency](/blog/how-to-run-a-transparent-company). Our product is open source and released under an MIT License. You can read [our source code](https://github.com/PostHog/posthog), raise your own issues and PRs, and [build apps atop PostHog](/blog/how-we-built-an-app-server) to support custom data flows. Anyone can read or comment on our [public roadmap](/roadmap).
+PostHog is [built on transparency](/blog/how-to-run-a-transparent-company). Our product is open source and released under an MIT License. You can read [our source code](https://github.com/PostHog/posthog), raise issues and PRs, build your own integrations, and vote on our [public roadmap](/roadmap). Got a question? Ask it in our [community forum](/questions).
 
-> This guide compares Mixpanel's self-serve Growth plan with PostHog Cloud. Features available on other plans are marked as such.
-
-## Feature comparison
+## Comparison PostHog and Mixpanel
 
 ### Platform
 
-<ComparisonTable column1="Mixpanel" column2="PostHog">
-  <ComparisonRow column1={true} column2={true} feature="Product analytics" description="Track events and conversion; analyze user behavior" />
-  <ComparisonRow column1={false} column2={true} feature="Session replay" description="Watch real users use your product; diagnose bugs" />
-  <ComparisonRow column1={false} column2={true} feature="Feature flags" description="Roll out features safely; toggle features for cohorts or individuals" />
-  <ComparisonRow column1="Enterprise" column2={true} feature="Experiments" description="Test changes and analyze their impact" />
-  <ComparisonRow column1={false} column2={true} feature="User surveys" description="Ask users for qualitative feedback and gather responses" />
-  <ComparisonRow column1={false} column2={true} feature="Heatmaps" description="Track where users click and why" />
-  <ComparisonRow column1={true} column2={true} feature="Event pipelines" description="Push and pull data to enrich customer profiles" />
-  <ComparisonRow column1={false} column2={true} feature="Open source" description="Build your own apps and contribute code" />
+Entering a credit card unlocks almost all our features and you **only pay for what you use**. You get a [generous free allowance](#pricing) _every_ month and you can set billing limits to prevent surprise bills.
```

Comment:
This feels a bit out of place so early. I think it would be good with pricing, but probably not needed at this point in the article.